src/game_interpreter.cpp

```diff
@@ -197,11 +198,12 @@ void Game_Interpreter::Update() {
 		}
 
 		if ((Game_Temp::battle_calling && !Game_Temp::battle_running) ||
-			Game_Temp::shop_calling ||
-			Game_Temp::name_calling ||
-			Game_Temp::menu_calling ||
-			Game_Temp::save_calling ||
-			Game_Temp::load_calling ||
+			IsShopCalling() ||
+			IsNameCalling() ||
+			IsMenuCalling() ||
+			IsSaveCalling() ||
+			IsLoadCalling() ||
+			Main_Data::game_data.party_location.menu_calling ||
```

Comment:
Not sure if we need the ESC menu calling flag here.

@Ghrabry what do you think about this?

Reply:
Fixed